src/status_im2/contexts/wallet/send/select_address/view.cljs

```diff
@@ -41,70 +44,155 @@
                        :container-style style/empty-container-style}]))
 
 (defn- address-input
```

Comment:
can be done outside this pr, I believe we can make this a common component as the same component is used on watch-only select address screen too 👍